generated/nidigital/nidigital_service.cpp

```diff
@@ -29,6 +29,15 @@ namespace nidigital_grpc {
       output->Add(item != VI_FALSE);
     }
   }
+  template <typename T1, typename T2>
```

Comment:
Make sure to regenerate all generated files once upstream branch is submitted to feature branch and merged back into this PR.

Reply:
regenerated after merge